R/backend-postgres.R

```diff
@@ -19,7 +19,7 @@
 #'
 #' lf <- lazy_frame(a = TRUE, b = 1, c = 2, d = "z", con = simulate_postgres())
 #' lf %>% summarise(x = sd(b, na.rm = TRUE))
-#' lf %>% summarise(y = cor(b, c), y = cov(b, c))
+#' lf %>% summarise(y = cor(b, c), z = cov(b, c))
```

Comment:
`summarise(df, y = ..., y = ...)` currently does not work correctly (also before this PR). I created an issue for this #785